config/src/snapshot.rs

```diff
@@ -0,0 +1,36 @@
+//! Configuration related to Snapshot specifically
+
+/// Functioning mode of the Snapshot Iroha module
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    Default,
+    strum::Display,
+    strum::EnumString,
+    serde_with::SerializeDisplay,
+    serde_with::DeserializeFromStr,
+)]
+#[strum(serialize_all = "snake_case")]
+pub enum Mode {
+    /// Read the snapshot on startup, update periodically
+    #[default]
+    Normal,
```

Comment:
```suggestion
    ReadWrite,
```

`Normal` seems ambiguous

Reply:
Updated (`read_write` in serialized form)